include/rocksdb/comparator.h

```diff
@@ -20,7 +21,7 @@ class Slice;
 // used as keys in an sstable or a database.  A Comparator implementation
 // must be thread-safe since rocksdb may invoke its methods concurrently
 // from multiple threads.
-class Comparator {
+class Comparator : public Customizable {
```

Comment:
`Customizable` increases the comparator size, what would be the performance impact?

Reply:
Here is a comparison of master and this branch for some operations from benchmark.sh (same performance tests as the performance Wiki):

Master:  
readrandom   :     461.911 micros/op 138552 ops/sec;   35.1 MB/s (7364222 of 11651999 found)
multireadrandom :     461.621 micros/op 138639 ops/sec; (7369984 of 11654990 found)
overwrite    :     694.754 micros/op 92117 ops/sec;   36.9 MB/s
readwhilewriting :     659.770 micros/op 97001 ops/sec;   30.8 MB/s (6496957 of 8195999 found)

Change: 
readrandom   :     461.858 micros/op 138568 ops/sec;   35.1 MB/s (7365481 of 11649999 found)
multireadrandom :     461.689 micros/op 138619 ops/sec; (7366962 of 11652990 found)
overwrite    :     698.994 micros/op 91558 ops/sec;   36.7 MB/s
readwhilewriting :     661.834 micros/op 96698 ops/sec;   30.7 MB/s (6455689 of 8151999 found)

This change does not appear to have a noticeable performance impact.